tests/unit/test_cost.py

```diff
@@ -0,0 +1,6 @@
+from gh_actions_exporter.cost import Cost
+
+
+def test_get_job_cost(workflow_job, settings):
+    assert Cost(settings).get_job_cost(workflow_job, "large") == 0.16
+    assert Cost(settings).get_job_cost(workflow_job, "xxx") == 0.08
```

Comment:
This is done on purpose. Basically the behavior is made so that if it's a self-hosted runner, we get the tag of the corresponding flavor and we compute with it. On the other hand if it is github-hosted, at this moment we return nothing for the flavor, so it puts by default a cost of 0.008. I don't know if it's clear, I can change it if you don't like it

Reply:
Ah I see, with the new logic you have implemented to calculate cost I don't think we should behave like this anymore. Because we can have a GitHub Larger hosted runner, which won't have the same cost as the default github hosted runner.

Here's what I suggest us to do:

Today we have the default settings of job cost like this:
```python
    job_costs: Optional[Dict[str, float]] = {
        "medium": 0.008,
        "large": 0.016,
        "xlarge": 0.032,
        "2xlarge": 0.064,
        "3xlarge": 0.128,
    }
    
```

Let's update the default settings to something around the lines of (which can be overwriten if needed, but it's fair to add those new label as default values):
```python
    job_costs: Optional[Dict[str, float]] = {
        "ubuntu-latest": 0.008,
        "ubuntu-22.04": 0.008,
        "ubuntu-20.04": 0.008,
        "medium": 0.008,
        "large": 0.016,
        "xlarge": 0.032,
        "2xlarge": 0.064,
        "3xlarge": 0.128,
    }
```    

Then apply the same logic for all, this will allow us to override the configuration, when needed, to handle the larger github hosted runners.

Reply:
I will add the prices for the large Ubuntu runners, is it okay with you if I take the prices [here](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#per-minute-rates)?